core/model/group.go

```diff
@@ -57,8 +57,8 @@ func (g *Group) IsCurrentUserAdmin(currentUserID string) bool {
 }
 
 // GetMembersAsNotificationRecipients Gets members as notification recipients
-func (g *Group) GetMembersAsNotificationRecipients(currentUserID string, subMembers []ToMember) []NotificationRecipient {
-	var recipients []NotificationRecipient
+func (g *Group) GetMembersAsNotificationRecipients(currentUserID string, subMembers []ToMember) []UserRef {
```

Comment:
Looks like we could probably get rid of this function altogether if it is unused.

Reply:
Nope. Actually your next comment was a really nice catch. Thanks for the below thought @shurwit!